index.html

```diff
@@ -937,20 +937,32 @@ <h2>
         </p>
       </section>
     </section>
-    <section class="informative">
+    <section>
       <h2>
         Privacy and Security Considerations
       </h2>
-      <section>
-        <h3>
-          Access to aspects of a user's local computing environment
-        </h3>
-        <p>
-          The screen orientation type and angle of the device can be accessed
-          with the API specified in this document, and can be a potential
-          fingerprinting vector.
-        </p>
-      </section>
+      <p>
+        A screen's [=current orientation type|type=] and [=current orientation
+        angle|angle=] are a potential fingerprinting vector. To resist
+        fingerprinting (e.g., in private browsing), user agents MAY:
+      </p>
+      <ol>
+        <li>Restrict the value return by the {{ScreenOrientation/type}}
+        attribute to {{OrientationType/"portrait-primary"}} or
+        {{OrientationType/"landscape-secondary"}} to match the screen's aspect
+        ratio.
+        </li>
+        <li>Always return `0` for the value of the {{ScreenOrientation/angle}}
+        attribute.
+        </li>
+        <li>If the screen orientation changes, only fire the
+          <a data-link-for="ScreenOrientation">change</a> event when the
+          [=current orientation type=] changes from [=portrait=] to
+          [=landscape=], or vice versa. This both protects user's privacy by
+          not revealing how a device is being held, and prevents using the
+          [=secondary=] orientation as a fingerprinting vector.
```

Comment:
Should this say "other" since above it wants to use "landscape-secondary"? Or is that a typo above?

Reply:
oh lol, yes, that was a typo above.